testsuite/features/step_definitions/setup_steps.rb

```diff
@@ -488,16 +488,31 @@
       log "The MU repository #{unique_repo_name} was already created, we will reuse it."
     else
       content_type = deb_host?(client) ? 'deb' : 'yum'
-      steps %(
-        When I follow the left menu "Software > Manage > Repositories"
-        And I follow "Create Repository"
-        And I enter "#{unique_repo_name}" as "label"
-        And I enter "#{repo_url.strip}" as "url"
-        And I select "#{content_type}" from "contenttype"
-        And I click on "Create Repository"
-        Then I should see a "Repository created successfully" text or "The repository label '#{unique_repo_name}' is already in use" text
-        And I should see "metadataSigned" as checked
-      )
+
+      if %w[slmicro60_minion slmicro61_minion].include?(client)
+        steps %(
+          When I follow the left menu "Software > Manage > Repositories"
+          And I follow "Create Repository"
+          And I enter "#{unique_repo_name}" as "label"
+          And I enter "#{repo_url.strip}" as "url"
+          And I select "#{content_type}" from "contenttype"
+          And I uncheck "metadataSigned"
+          And I click on "Create Repository"
+          Then I should see a "Repository created successfully" text or "The repository label '#{unique_repo_name}' is already in use" text
+          Then I should see "metadataSigned" as unchecked
+        )
+      else
+        steps %(
+          When I follow the left menu "Software > Manage > Repositories"
+          And I follow "Create Repository"
+          And I enter "#{unique_repo_name}" as "label"
+          And I enter "#{repo_url.strip}" as "url"
+          And I select "#{content_type}" from "contenttype"
+          And I click on "Create Repository"
+          Then I should see a "Repository created successfully" text or "The repository label '#{unique_repo_name}' is already in use" text
+          Then I should see "metadataSigned" as checked
+        )
+      end
```

Comment:
Alternative proposal:

```suggestion
      node = get_target(client)
      if node.os_family.include?('sl-micro') && node.os_version.include?('6')
        metadata_signed_check = 'uncheck'
      else
        metadata_signed_check = 'check'
      end
      steps %(
        When I follow the left menu "Software > Manage > Repositories"
        And I follow "Create Repository"
        And I enter "#{unique_repo_name}" as "label"
        And I enter "#{repo_url.strip}" as "url"
        And I select "#{content_type}" from "contenttype"
        And I #{metadata_signed_check} "metadataSigned"
        And I click on "Create Repository"
        Then I should see a "Repository created successfully" text or "The repository label '#{unique_repo_name}' is already in use" text
      )
```

Reply:
```
        And I #{metadata_signed_check} "metadataSigned"
```
apart from not being readable, is not English text. Gherkin text should remain English. Don't generate code out of variables.